descartes_core/include/descartes_core/cart_trajectory_pt.h

```diff
@@ -70,8 +70,8 @@ struct ToleranceBase
     #param tol Total tolerance zone (assumed symetric about nominal)
     */
   template<typename T>
-  static T createSymmetric(const double x, const double y, const double z,
-                                          const double tol)
+  static T createSymmetric(const double& x, const double& y, const double& z,
```

Comment:
Do we need to pass these values by reference?  Are they changed by the createSymmetric method?